packages/tests/src.ts/test-wallet.spec.ts

```diff
@@ -4,14 +4,57 @@ import assert from "assert";
 
 import  {  ethers } from "ethers";
 import { loadTests, TestCase } from "@ethersproject/testcases";
+import { BytesLike } from "@ethersproject/bytes";
 import * as utils from './utils';
 import { arrayify, getAddressFromAccount } from "ethers/lib/utils";
 import {
+    AccountCreateTransaction, AccountId,
+    Client,
     ContractCreateTransaction,
     ContractExecuteTransaction,
     FileAppendTransaction,
-    FileCreateTransaction, PublicKey, Transaction
+    FileCreateTransaction, Hbar, PrivateKey, PublicKey, Transaction, TransactionId,
+    Key as HederaKey
 } from "@hashgraph/sdk";
+import { Wallet } from "./../../wallet";
+import { Key } from "@hashgraph/proto";
+import { BaseProvider } from "./../../providers";
+
+/**
+ * Helper function that returns a Wallet instance from the provided ED25519 credentials,
+ * provided from portal.hedera.com
+ * @param account
+ * @param provider
+ */
+const createWalletFromED25519 = async (account: any, provider: BaseProvider) => {
```

Comment:
We should find a way to avoid spamming testnet with many useless accounts ( created in our unit tests) . I'd suggest we switch to `previewnet` for unit testing, as it's cleaned up sometimes. (this may cause test flakyness tho). 

This is not a comment about the PR, but a general thought.

Reply:
Changed it to `previewnet`